Firestore/core/src/model/database_id.cc

```diff
@@ -34,6 +36,15 @@ DatabaseId::DatabaseId(std::string project_id, std::string database_id) {
   rep_ = std::make_shared<Rep>(std::move(project_id), std::move(database_id));
 }
 
+/* static */
+DatabaseId DatabaseId::FromName(absl::string_view name) {
+  ResourcePath resource_name = ResourcePath::FromString(name);
```

Comment:
Nit: consider using `auto`.

Reply:
Removed (we have these in many places though)